### ISSUE TITLE
HACS Installation Errors

### ISSUE THREAD

**Issue by carltonwb** [hacs/integration#1106]
## Version of HACS

0.24.3

## Describe the bug
After following the installation instructions and using the UI to install HACS, it will not show in the sidebar, and I have no access to setup the integrations/plugins.

## Debug log

<details>

```
`2020-04-30 07:55:51 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry  for hacs
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 216, in async_setup
    hass, self
  File "/config/custom_components/hacs/__init__.py", line 85, in async_setup_entry
    startup_result = await hacs_startup()
  File "/config/custom_components/hacs/__init__.py", line 136, in hacs_startup
    lovelace_info = await system_health_info(hacs.hass)`
```

</details>

**Comment by sjbakerboy** [comment]
Same issue!

**Comment by VarenDerpsAround** [comment]
Same Issue! Tried rebooting, and a full host reboot. Glad to know it wasn't just me.

**Comment by ludeeus** [comment]
@VarenDerpsAround @sjbakerboy GOOD!
Then _any_ of you can get the logs that are missing in this issue.

**Comment by Marxcel** [comment]
I have the same issue as well.

**Comment by VarenDerpsAround** [comment]
I figured out (maybe just my issue in this circumstance) but the issue I was having. It seems my personal access token expired with this update? I did not have the "component folder" get wiped like another user did, however I removed the integration, recreated a new access token, pasted it in and it came back to life like nothing was ever wrong.

I hereby recind my "same issue" comment

Ludeeus you're a boss, don't forget it. I was in the process of enabling logs when I found that the token had expired. Thanks, I'll still follow along with the class.

**Comment by sjbakerboy** [comment]
Ok not sure if there is an issue with Samba share and Mac OS, but I have solved the issue now following the exact steps outlined on GItHub for installation using a windows based machine.

I think there may have been a corruption when copying the unzipped hacs folder to the custom_components folder. I ended up restoring my whole home assistant install on my raspberry pi. Tried again on MacOS and same issue.

Finally tried the windows option and worked first time!

Hope this helps someone else!
